src/sentry/tasks/servicehooks.py

```diff
@@ -1,13 +1,96 @@
 from __future__ import absolute_import, print_function
 
 import six
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 from time import time
 
+from sentry.api.serializers import serialize, app_platform_event
 from sentry.http import safe_urlopen
+from sentry.models import Group, SentryAppInstallation, ServiceHook
 from sentry.tasks.base import instrumented_task
 from sentry.utils import json
 from sentry.utils.http import absolute_uri
 
+# This is an extra, explicit, measure to ensure we only send events for
+# resource changes we deem necessary.
+ALLOWED_ACTIONS = (
+    'issue.created',
+)
+
+# We call some models by a different name, publically, than their class name.
+# For example the model Group is called "Issue" in the UI. We want the Service
+# Hook events to match what we externally call these primitives.
+RESOURCE_RENAMES = {
+    'group': 'issue',
+}
+
+
+@receiver(post_save, sender=Group)
+def resource_changed(sender, instance, created, **kwargs):
+    process_resource_change.delay(sender, instance.id, created)
```

Comment:
right now we're queueing a task for every group change.. could we queue tasks only when we know something needs to be done? i.e. move some of the logic from process_resource_changed to this method. (I'm assuming `delay` is what puts it in the queue).

Reply:
For Group, this probably doesn't matter since it's created out of band of any request an actual person is making, but for any other resource in the future, I think it's probably better to do less during the lifespan of the request and offload as much as possible to a background job. No reason to hold a user up for something that's not immediately necessary, imo.